### PR TITLE
Code Style Clean Up and Removal of Debug Function

### PR DESCRIPTION
**Pull request by bartboy011**
This PR raises the repo to PSR-2 standards, with a few exceptions (no namespacing, no using non-camelcase method names). Additionally, it removes the `dbg()` method and all instances of `if($this->debug)`. Finally, it adds linting to the repo via PHP_Codesniffer.

### CODE REVIEW

**Comment by bartboy011** [comment]
@ActiveCampaign/integration

File: includes/test.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?php
```
Comment (cristiangrama) [inline includes/test.php]: @bartboy011 This could be removed.

Comment (bartboy011) [reply includes/test.php]: @cristiangrama Good catch, that was an accidental commit

**Comment by mthommes** [comment]
@bartboy011 @cristiangrama The `debug` feature is useful for support troubleshooting. Is there an alternative & better way of doing this? If so I think we should document that somehow.

**Comment by bartboy011** [comment]
@cristiangrama @mthommes  This should be ready to go for a review now

**Review by mthommes — APPROVED** [review]

**Comment by bartboy011** [comment]
@cristiangrama This was approved (not sure you get the notifications)

File: includes/Tracking.class.php [hunk 2]
```diff
@@ -1,118 +1,125 @@
...
-		}
-		$response = $this->curl($request_url, array(), "GET", "tracking_event_list");
-		return $response;
-	}
+    /*
+     * Returns existing tracked events.
+     */
+    public function event_list($params)
+    {
+        if ($this->version == 1) {
```
Comment (cristiangrama) [inline includes/Tracking.class.php:61]: @bartboy011 Can you do a follow up PR to fix this?

**Review by cristiangrama — APPROVED** [review]

File: includes/Tracking.class.php (hunk 2 above)
Comment (bartboy011) [reply includes/Tracking.class.php:61]: 👍

**Pull request merged**